app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :comment, :rating
+
+  belongs_to :user, if: :current_user?
+
+  def current_user?
+    scope == object.user
+  end
```

Comment:
Concerning lines 4 - 8, if we aren't making use of the `current_user?` method we can go ahead and remove it for now and include it when we put up a PR that will make use of the current user

Reply:
Removed current_user? for now.